dotnet/src/Functions/Functions.OpenAPI/Extensions/KernelAIPluginExtensions.cs

```diff
@@ -15,9 +15,13 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.SemanticKernel.Diagnostics;
+using Microsoft.SemanticKernel.Functions.OpenAPI.Authentication;
 using Microsoft.SemanticKernel.Functions.OpenAPI.Model;
 using Microsoft.SemanticKernel.Functions.OpenAPI.OpenApi;
 using Microsoft.SemanticKernel.Orchestration;
+using Newtonsoft.Json;
```

Comment:
Would it be possible to use `System.Text.Json` for this functionality?

Reply:
yes, updated